db/seeds.rb

```diff
@@ -6,24 +6,32 @@
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
 
-18.times do |t|
-  Kitchen.create(floor: t)
+puts 'Creating kitchens 3 to 18'
+16.times do |t|
+  Kitchen.create(floor: t+3 )
 end
 
-basic_applaince_names = ['sütő', 'mikró']
+
+min_appliance_per_kitchen = 2
+max_appliance_per_kitchen = 10
+
+min_comment_per_appliance = 3
+max_comment_per_appliance = 20
 
 Kitchen.all.each do |kitchen|
-  basic_applaince_names.each do |appliance_name|
-    appliance = Appliance.create(name:        appliance_name,
-                                 category:    Appliance.categories.values.sample,
-                                 status:      Appliance.statuses.values.sample,
-                                 description: "Ez egy eszköz leírása.",
-                                 kitchen:     kitchen)
-    10.times do |t|
-      ApplianceComment.create(body:      'Komment teste',
-                              category:  ApplianceComment.categories.values.sample,
+  Faker::Number.between(from: min_appliance_per_kitchen , to: max_appliance_per_kitchen).times do |num|
+    appliance = Appliance.create(name: "#{Faker::Appliance.brand} #{Faker::Appliance.equipment}",
+                                 category: Appliance.categories.values.sample,
```

Comment:
What do you think about aligning the create parameter hash values that they start at the same column?
I think its more readable.
RubyMine can be configured to align it like that on formatting.